<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UAVMissionControlWidget</class>
 <widget class="QTabWidget" name="UAVMissionControlWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tabMissionControl">
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="title">
    <string notr="true">Mission Parameters</string>
   </attribute>
   <widget class="QLabel" name="labelNumUAV">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of UAVs</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxNumUAV">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>53</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelRole">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Role</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEquipment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UAV Equipment</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxEquipment">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>85</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>UAV0</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelMissionType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mission Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxMissionType">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Mapping</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Surveillance</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Disaster Management</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Mission</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetEquipment">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <item>
     <property name="text">
      <string>Camera</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Infrared Sensor</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom Payload</string>
     </property>
    </item>
   </widget>
   <widget class="QScrollArea" name="scrollAreaMissionOverview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>605</width>
       <height>155</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="tableWidgetMissionOverview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>141</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>580</width>
        <height>100</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <column>
       <property name="text">
        <string>UAV</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Equipment</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Role</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelMissionOverviewTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mission Overview::</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMissionOverview">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UNDEF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonArea">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>95</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Define Area</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAreaClear">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>95</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Area</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRole">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tabRealTime">
   <attribute name="title">
    <string notr="true">Real Time Configuration</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
